samples/getWorkloadStatus.py

```diff
@@ -2,60 +2,118 @@
+        self.processor_events = []
+        self.processor_fork_events = []
+        self.job_consolidation_status_events = []
+        self.job_processing_status_events = []
 
-    def __init__(self, event_name, time_stamp, contents):
-        self.event_name = event_name
-        self.time_stamp = time_stamp
-        self.contents = contents
+    def get_events(self):
+        print "Getting Relevant Workload Events"
+        while True:
+            # Setting message visibility to 2 days, so that the message won't get processed again
+            messages = self.log_queue_service.get_messages(self.config.workload_tracker_queue_name, 32)
+            for msg in messages:
+                parsed = json.loads(msg.content)
+                event = WorkloadEvent(WorkloadEventType(parsed["event_type"]), msg.insertion_time, parsed["content"])  
 
-if __name__ == "__main__":
-    config = Config()
+                # Workload Completed
+                if event.event_type == WorkloadEventType.WORKLOAD_DONE:
+                    self.workload_complete_event = event
 
-    job_queue_service = QueueService(account_name=config.storage_account_name,
-                sas_token=config.job_status_queue_sas_token)
-    
-    log_queue_service = QueueService(account_name=config.storage_account_name,
-                sas_token=config.logger_queue_sas)
-
-    while True:
-        messages = log_queue_service.get_messages(config.logger_queue_name, 32)
-        if len(messages) == 0:
-            break
-
-        for msg in messages:
-            for event in LOG_EVENT_MSGS:
-                if event in str(msg.content):
-                    print str(msg.insertion_time) + " " + msg.content
+                # Scheduler - Main Start
+                if event.event_type == WorkloadEventType.SCHEDULER_START:
+                    self.scheduler_start_event = event
+                
+                # Processor - Main Start
+                if event.event_type == WorkloadEventType.PROCESSOR_START:
+                    self.processor_events.append(event)
+                
+                # Processor - Fork Start
+                if event.event_type == WorkloadEventType.PROCESSOR_FORK_START:
+                    self.processor_fork_events.append(event)
+                
+                # Job Consolidation Status
+                if event.event_type == WorkloadEventType.WORKLOAD_CONSOLIDATION_STATUS:
+                    self.job_consolidation_status_events.append(event)
                 
+                # Job Processing Status
+                if event.event_type == WorkloadEventType.WORKLOAD_PROCESSING_STATUS:
+                    self.job_processing_status_events.append(event)
+                
+                # All Jobs Queued
+                if event.event_type == WorkloadEventType.JOBS_QUEUE_DONE:
+                    self.jobs_queued_done = event
+
+                if event is not None:
+                    print str(event.timestamp) + " " + event.contents
```

Comment:
Where does this print go?

Reply:
This script gets run outside of the Azure topology. It is for monitoring the workload, so print just goes to the console that is running this.